Package.swift

```diff
@@ -1050,8 +1063,7 @@ if ProcessInfo.processInfo.environment["ENABLE_APPLE_PRODUCT_TYPES"] == "1" {
     }
 }
 
-if ProcessInfo.processInfo.environment["SWIFTPM_SWBUILD_FRAMEWORK"] == nil &&
-    ProcessInfo.processInfo.environment["SWIFTPM_NO_SWBUILD_DEPENDENCY"] == nil {
+if !shoudUseSwiftBuildFramework {
```

Comment:
Recovering `&& ProcessInfo.processInfo.environment["SWIFTPM_NO_SWBUILD_DEPENDENCY"] == nil` seems to allow the previous behaviour of disabling swift-build completely from SwiftPM.

Reply:
The `SWIFTPM_NO_SWBUILD_DEPENDENCY` was a temporary workaround until we were able to properly integration SwiftBuild in the toolchain build.  I don't believe it's required anymore.